sources/Application/Views/ModalDialogs/PagedImportSampleDialog.cpp

```diff
@@ -7,10 +7,7 @@
 #define LIST_SIZE 15
```

Comment:
LIST_SIZE can be increased if we removed the buttons

Reply:
Yes I was tempted to do this but wan't sure if I should, so cool, will do! 👍🏻